browser/brave_content_browser_client.cc

```diff
@@ -109,9 +123,9 @@ void BraveContentBrowserClient::BrowserURLHandlerCreated(
                           content::BrowserURLHandler::null_handler());
   handler->AddHandlerPair(&webtorrent::HandleTorrentURLRewrite,
                           &webtorrent::HandleTorrentURLReverseRewrite);
+  ChromeContentBrowserClient::BrowserURLHandlerCreated(handler);
```

Comment:
This point is what I worried.
When user types chrome url, load to url is initialized with brave url.
Then, I think brave url is the input of URLHandler before it is re-written our handler by below.
If so, that could cause skipping upstream URLHandler that checks chrome scheme.

Reply:
I see your point, but we may need to add two handers then because we have to run after ChromeContentBrowserClient to fix kChromeUISyncInternalsHost. I'll look at the upstream code to try to find something it breaks and update either way to convert back to chrome before the superclass method